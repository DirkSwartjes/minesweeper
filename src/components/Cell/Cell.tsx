import classes from './cell.module.css';

const Cell: React.FC = () => (
  <div className={classes.cell} />
);

export default Cell;
