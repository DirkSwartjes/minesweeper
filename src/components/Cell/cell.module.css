.cell {
  width: 20px;
  height: 20px;
  border: 2px solid;
  border-color: #f5f5f5 #808080 #808080 #f5f5f5;
  background-color: #c0c0c0;
}