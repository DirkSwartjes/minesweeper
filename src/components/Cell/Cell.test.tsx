import { render } from '@testing-library/react';
import Cell from './Cell';

describe('Cell component', () => {
  it('renders correctly', () => {
    const {container} = render(<Cell />);

    expect(container).toMatchSnapshot();
  });
});