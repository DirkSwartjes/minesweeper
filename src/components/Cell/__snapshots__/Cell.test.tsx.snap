// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Cell component > renders correctly 1`] = `
<div>
  <div
    class="_cell_f9220d"
  />
</div>
`;
